images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -224,16 +224,9 @@ function Get-DotnetRuntimes {
     }
 }
 
-function Get-DotnetFrameworkTools {
-    $path = "${env:ProgramFiles(x86)}\Microsoft SDKs\Windows\*\*\NETFX*"
-    Get-ChildItem -Path $path -Directory | Group-Object {
-        $_.Fullname -Replace " \d+\.\d+(\.\d+)?", " <version>"
-    } | ForEach-Object {
-        [PSCustomObject]@{
-            Versions =  $_.Group.Name | ForEach-Object { $_.Split(" ")[1] }
-            Path = $_.Name
-        }
-    }
+function Get-DotnetFrameworkVersions {
+    $path = "${env:ProgramFiles(x86)}\Microsoft SDKs\Windows\*\*\NETFX * Tools"
+    Get-ChildItem -Path $path -Directory | ForEach-Object { $_.Name | Take-Part -Part 1 } -join ' '
```

Comment:
I think this would rather look like:
`(Get-ChildItem -Path $path -Directory | ForEach-Object { $_.Name | Take-Part -Part 1 }) -join ' '`
to make result like 1.1.1 2.1.1 3.1.1 ?

Reply:
Fixed. Good catch